src/main/webapp/detailsScript.js

```diff
@@ -0,0 +1,104 @@
+let API_KEY;
+// Obtain the API_KEY from local file.
+fetch('apiKey.json')
+  .then(response => response.json())
+  .then(jsonResponse => {
+    API_KEY = jsonResponse.apiKey;
+    getBookDetails();
+  });
+
+// To get results from UK publishing houses.
+const country = 'UK';
+
+// Obtain the book ID from the query string. This is the Google Books API ID.
+let id = parseQueryString(window.location.search).id;
```

Comment:
This only seems useful in initializeViewer. Why is this not a local vairable in that function?

Reply:
It it also used to fetch data from Books API to get info about the specific book